windows/ReactNativePSPDFKit/ReactNativePSPDFKit/PSPDFKitModule.cs

```diff
@@ -0,0 +1,85 @@
+﻿using PSPDFKit;
+using ReactNative.Bridge;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ReactNativePSPDFKit
+{
+    /// <summary>
+    /// A bridge to the PSPDFKit SDK.
+    /// </summary>
+    class PSPDFKitModule : ReactContextNativeModuleBase
+    {
+        private API _API;
+        private string VERSION_KEY = "versionString";
+
+        public PSPDFKitModule(ReactContext reactContext, API api) : base(reactContext)
+        {
+            _API = api;
+        }
+
+        /// <summary>
+        /// Open the native file picker for the user to select a pdf.
+        /// </summary>
+        [ReactMethod]
+        public void OpenFile()
+        {
+            DispatcherHelpers.RunOnDispatcher(async () =>
+            {
+                var file = await PickPDF();
+                if (file != null)
+                {
+                    LoadViaAPI(file);
+                }
+            });
+        }
+
+        private async Task<Windows.Storage.StorageFile> PickPDF()
+        {
+            var picker = new Windows.Storage.Pickers.FileOpenPicker
+            {
+                ViewMode = Windows.Storage.Pickers.PickerViewMode.Thumbnail,
+                SuggestedStartLocation = Windows.Storage.Pickers.PickerLocationId.DocumentsLibrary
+            };
+            picker.FileTypeFilter.Add(".pdf");
+
+            return await picker.PickSingleFileAsync();
+        }
+
+        private async void LoadViaAPI(Windows.Storage.StorageFile file)
+        {
+            if (file == null) return;
+
+            await _API.OpenAsync(file);
```

Comment:
You can handle exceptions like this...

`using Windows.UI.Popups;`

```csharp
            try
            {
                await _API.OpenAsync(file);
            }
            catch (Exception e)
            {
                var dialog = new MessageDialog(e.Message);
                await dialog.ShowAsync();
            }
```

Reply:
Nice. That's much nicer.